spec/system/referee_interface/referee_can_submit_a_reference_for_candidate_spec.rb

```diff
@@ -42,9 +44,11 @@ def when_the_candidate_submits_the_application
   def then_i_receive_an_email_with_a_magic_link
     open_email('terri@example.com')
 
-    reference_feedback_url = get_reference_feedback_url
+    matches = current_email.body.match(/(http:\/\/localhost:3000\/reference\?token=[\w-]{20})/)
+    @token = Rack::Utils.parse_query(URI(matches.captures.first).query)['token']
```

Comment:
What's this for? 🤔

Reply:
Twas part of refactoring `get_reference_feedback_url` method. It's used in:

```ruby
def then_i_see_the_confirmation_page
  expect(page).to have_current_path(referee_interface_confirmation_path(token: @token))
end
```